emap-setup/emap_runner/validation/validation_runner.py

```diff
@@ -195,8 +198,8 @@ def _exceeded_timeout(self) -> bool:
 
     @property
     def _elapsed_time(self) -> timedelta:
-        """Seconds elapsed since the runner started"""
-        return timedelta(microseconds=time() - self.start_time)
+        """Time elapsed since the runner started"""
+        return timedelta(seconds=time() - self.start_time)
```

Comment:
🙃

Probably best to change the type of `start_time` to `datetime.datetime` then
```suggestion
        """Time elapsed since the runner started"""
        assert self.start_time is not None
        return datetime.now() - self.start_time
```
?

Reply:
Since `start_time` isn't needed outside that method I've turned it into a local. And it should be a monotonic clock to ignore system time jumps.